bot-twitter/src/variables.ts

```diff
@@ -0,0 +1,4 @@
+export const ETHEREUM_NETWORK = process.env.ETHEREUM_NETWORK || 'kovan';
+export const KEEPER_MIN_PROFIT_DAI = process.env.KEEPER_MIN_PROFIT_DAI
+    ? parseInt(process.env.KEEPER_MIN_PROFIT_DAI)
+    : undefined;
```

Comment:
And what about NaN? I would better parse it anyway and then check for `Number.isNaN` during runtime:
```suggestion
export const KEEPER_MIN_PROFIT_DAI = parseInt(process.env.KEEPER_MIN_PROFIT_DAI || '');
```